examples/demo.py

```diff
@@ -25,7 +25,6 @@
         oles=True,
         calc_gm_score=True,
         is_arch_attention_enabled=True,
-        is_regularization_enabled=True,
```

Comment:
I think this might be outdated for drnas config. We need to add l2 as default reg_type as well in the drnas_config